src/radical/pilot/agent/agent_0.py

```diff
@@ -340,77 +350,59 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info("Starting the agent services")
+        service_descriptions = self._cfg.services
+        services = list()
+
+        for service_desc in service_descriptions:
+            task = dict()
+            task['origin'] = 'services'
+            task['description'] = TaskDescription(service_desc).as_dict()
+            task['state'] = rps.AGENT_STAGING_INPUT_PENDING
+            task['status'] = 'NEW'
+            task['type'] = 'task'
+            task['uid'] = ru.generate_id('services.%(item_counter)04d',
+                                             ru.ID_CUSTOM, ns="TODO_WHATTOUSE")
+            self._service_task_ids.append(task['uid'])
+            task['pilot_sandbox'] = self._cfg.pilot_sandbox
+            task['task_sandbox'] = self._cfg.pilot_sandbox + task['uid'] + '/'
+            task['task_sandbox_path'] = self._cfg.pilot_sandbox + task['uid'] + '/'
+            task['session_sandbox'] = self._cfg.session_sandbox
+            task['resource_sandbox'] = self._cfg.resource_sandbox
+            task['pilot'] = self._cfg.pid
+            # task['resources'] = {'cpu': td['ranks'] *
```

Comment:
Resources are needed for post-mortem statistics - why were they commented out?

Reply:
they use td instance to get those params, I don't know where to get those. 

```{'cpu': 1, 'gpu': 0}, ```
this is `resources` key's value  when I don't add myself

Reply:
Well, you can use the service's task description, right?  Or am I missing something?

Reply:
So should I add these values in my examples/agent_services.py

Reply:
Default values should kick in if they are not specified in the original  service description.